helm/hpcc/values.yaml

```diff
@@ -621,20 +621,30 @@ roxie:
   ## replicas indicates the number of replicas per channel
   replicas: 2
   numChannels: 2
-  ## Set serverReplicas to indicate a separate replicaSet of roxie servers, with agent nodes not acting as servers
-  serverReplicas: 0
   ## Set localAgent to true for a scalable cluster of "single-node" roxie servers, each implementing all channels locally
   localAgent: false
   ## Adjust traceLevel to taste (1 is default)
   traceLevel: 1
+  ## set totalMemoryLimit to indicate how much memory is preallocated for roxie row data
+  totalMemoryLimit: "1Gi"
   ## Set mtuPayload to the maximum amount of data Roxie will put in a single packet. This should be just less than the system MTU. Default is 1400
   # mtuPayload: 3800
-  #channelResources:
-  #  cpu: "4"
-  #  memory: "4G"
-  #serverResources:
+
+  ## resources specifies the resources required by each agent pod
+  #resources:
   #  cpu: "1"
   #  memory: "1G"
+
+  ## Set serverReplicas to indicate a separate replicaSet of roxie servers, with agent pods not acting as servers
+  serverReplicas: 0
+  ## If serverReplicas is set, the resources required for the server pods can be configured separately from the agent (channel) pods
```

Comment:
minor: either "resources" should be defined, or both "serverResources" and "channelResources" correct?
For clarity/enforcement you could put in a oneOf, to ensure not all are defined.
I did that for alternate remote plane specifications: https://github.com/hpcc-systems/HPCC-Platform/blob/0bed23d313ac1f7396a0605b5e44f27cbaeb23bb/helm/hpcc/values.schema.json#L578

Reply:
It would be reasonable to set resources, then override one or other of the other two... Defining all three would mean that resources was not used, but I'm not sure I see that as a big problem